cmd/cluster.go

```diff
@@ -14,45 +14,43 @@ import (
 )
 
 func ClusterCommand() cli.Command {
-	clusterUpFlags := []cli.Flag{
+	clusterUpDownUpgradeFlags := []cli.Flag{
 		cli.StringFlag{
 			Name:   "cluster-file",
 			Usage:  "Specify an alternate cluster YAML file",
 			Value:  cluster.DefaultClusterConfig,
 			EnvVar: "CLUSTER_FILE",
 		},
 	}
-	clusterUpgradeFlags := []cli.Flag{
-		cli.StringFlag{
-			Name:   "cluster-file",
-			Usage:  "Specify an upgraded cluster YAML file",
-			Value:  "cluster.yml",
-			EnvVar: "CLUSTER_FILE",
-		},
-	}
 	return cli.Command{
 		Name:      "cluster",
 		ShortName: "cluster",
 		Usage:     "Operations on the cluster",
-		Flags:     clusterUpFlags,
+		Flags:     clusterUpDownUpgradeFlags,
 		Subcommands: []cli.Command{
 			cli.Command{
 				Name:   "up",
 				Usage:  "Bring the cluster up",
 				Action: clusterUpFromCli,
-				Flags:  clusterUpFlags,
+				Flags:  clusterUpDownUpgradeFlags,
+			},
+			cli.Command{
+				Name:   "down",
```

Comment:
We should ask the user for confirmation before removal..We also should provide a --force option to skip the confirmation and allow for automation and shoot-self-in-the-foot needs..

Reply:
I added force flag